test/unit/npm-dependncies.spec.js

```diff
@@ -0,0 +1,58 @@
+const npmDependencies = require("../../src/dynamicTypes/npmDependencies");
+
+describe("npm dependencies", () => {
+  describe("should load types for npm modules", () => {
+    it("get default type when request raw", () => {
+      const rawNpmType = npmDependencies.getRaw("react");
+      expect(rawNpmType).toMatchSnapshot();
+    });
+
+    it("get default type when request files and the namespaces is not @wix", async () => {
+      const npmDtsFiles = await npmDependencies.getFiles({
+        react: "16.8.6",
+        "react-dom": "16.8.6"
+      });
+      expect(npmDtsFiles).toMatchSnapshot();
+    });
+
+    it("get proper type for @wix scoped npm module", async () => {
+      global.fetch = jest.fn(() =>
+        Promise.resolve({
+          ok: true,
+          text: () => "type SababaType = {}"
+        })
+      );
+      const npmDtsFiles = await npmDependencies.getFiles({
+        "@wix/react-velo": "1.0.0"
+      });
+      expect(npmDtsFiles).toMatchSnapshot();
```

Comment:
this is why the snapshot looks weird for me. from reading the test you don't know what to expect. u can't understand what needs to be the result. I prefer asserts like u did in the following line